docs/_guide/introduction.md

```diff
@@ -11,7 +11,7 @@ GitHub's first page interactions were written using jQuery, which was widely use
 
 Rather than moving to entirely new paradigms, GitHub continued to use the same concepts within jQuery. Event Delegation was still heavily used, as well as querySelector. The event delegation concept was also extended to "element delegation" - discovering when Elements were added to the DOM, using the [Selector Observer](https://github.com/josh/selector-observer) library.
 
-From this emerged a set of patterns which were reduced down to their first principles. _Observing_ elements on the page, _listening_ to the events these elements or their children emit, and _querying_ the children of an element to mutate or extend them.
+From this emerged set of patterns which were reduced down to their first principles. _Observing_ elements on the page, _listening_ to the events these elements or their children emit, and _querying_ the children of an element to mutate or extend them.
```

Comment:
Thank you for working to improve the Catalyst docs! The existing `a` was not a typo, but I agree this phrase could be refined for clarity. What do you think of this change?
```suggestion
These patterns were reduced to first principles: _Observing_ elements on the page, _listening_ to the events these elements or their children emit, and _querying_ the children of an element to mutate or extend them.
```

Reply:
Sorry I misread the sentence. Your suggestion looks nice to me.